scripts/quarantine.yaml

```diff
@@ -0,0 +1,18 @@
+# The configurations resulting as a product of scenarios and platforms
```

Comment:
Would it be possible to make separate quarantine yamls per application/sample (in similar way as it's done for sample.yaml)?
In that way the solution would be more scalable and easier to maintain. In general applications and samples are independent. They could be maintained by various teams too.

Reply:
Yes. There is some flexibility to the quarantine. This time I put both scenarios in a single quarantine instance, because they both relate to the same issue. You can check an example at https://docs.zephyrproject.org/latest/guides/test/twister.html#quarantine

Reply:
sorry, I got you wrong. Creating separate yamls would be hard to maintain IMO. I preferer if there is a single file, and CI team and test leads have maintainership over. This will make it more transparent and traceable.